WebRoot/skins/_base/base4/skin.properties

```diff
@@ -2686,3 +2686,26 @@ OrganizerDialogInputText        = @DialogInputField@;
 DialogPropertySheet             = @FullWidth@;
 DialogPropertySheetCell         = padding:@SkinViewInnerSpacing@ 0;
 DialogPropertySheetCellLabel    = padding-bottom:0;
+
+#######################################################################
+#   Find locations/equipment dialog
+#######################################################################
+
+AttendeePickerDialog                    = 
+AttendeePickerDialogSection             = margin-bottom:@SkinWrapperPadding@;
+AttendeePickerChooserList               = margin:0; padding:0;
+AttendeePickerSectionTitle              = @Text@; margin:0; padding:0; font-weight:normal; margin-bottom:@SkinViewInnerSpacing@;
```

Comment:
are we trying to override some style using font-weight: normal, because that is default

Reply:
This rule targets `h6` element. And by default it looks bolder.